web/packages/teleterm/src/ui/DocumentAccessRequests/NewRequest/useNewRequest.ts

```diff
@@ -211,6 +211,12 @@ export default function useNewRequest(rootCluster: Cluster) {
       return;
     }
 
+    if (kind === 'namespace') {
+      // It is not possible to request a kube namespace through this function.
+      // The type should be corrected.
```

Comment:
Is it actually possible to adjust the type? I don't think so, as AFAIK we can't just remove `namespace` from `ResourceKind`. I'm not sure how much sense would it make to use `Exclude<ResourceKind, 'namespace'>` in this particular example.

Perhaps it'd be best to add a comment to `addOrRemoveResource` explaining that it's used for adding individual resources from the unified resources view and then this comment here could simply say that it's impossible to add kube namespaces straight from the unified resources view, so this function returns early?